R/read_extract_cmh.R

```diff
@@ -5,6 +5,11 @@
 #' @export
 #'
 read_extract_cmh <- function(year, file_path = get_boxi_extract_path(year = year, type = "CMH")) {
+  # Specify years available for running
+  if (year >= 1617 & year <= 2021) {
+    return(NULL)
+  }
```

Comment:
This should be a *NOT* `!`. Also I think it's better to specify the years explicitly for CMH (and DN) as they are now fixed and won't change.

```suggestion
  if (!(year %in% c("1617", "1718", "1819", "1920", "2021"))) {
    return(NULL)
  }
```

I think `year` should be string? Please change to integers if not (`1617L` etc.)